controllers/lectureController.js

```diff
@@ -0,0 +1,89 @@
+var formidable = require("formidable");
+const fs = require("fs");
+var dir = __dirname + '/../uploads/lectures/';
```

Comment:
Please prefer using node's 'path' module. Instead of string concatenation, use `path.join(__dirname, '...');`

Reply:
Done, Thank you for the suggestion